File: 프로그래머스/1/131705. 삼총사/삼총사.c
#include <stdio.h>
#include <stdbool.h>
#include <stdlib.h>

// number_len은 배열 number의 길이입니다.
int solution(int number[], size_t number_len) {
    int answer = 0;
    
    for(int i = 0; i < number_len - 2; i++){
        for(int j = i + 1; j < number_len - 1; j++){
            for(int l = j + 1; l < number_len; l++){
                if(number[i] + number[j] + number[l] == 0) answer++;
            }
        }
    }
    
    return answer;
}

File: 프로그래머스/1/131705. 삼총사/삼총사.py
def solution(number):
    answer = 0
    
    for i in range(len(number)):
        for j in range(i + 1, len(number)):
            for l in range(j + 1, len(number)):
                answer += 1 if number[i] + number[j] + number[l] == 0 else 0

    return answer